ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_layer1.schema.yml

```diff
@@ -27,3 +27,6 @@ keys:
           communication:
             type: bool
             description: "Enable transceiver SMBus fail and reset logging."
+          dom_and_communication:
```

Comment:
i believe the key name should be just `enable`

Reply:
Fixed